<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-V6QH7CG" failures="1" tests="1" name="Test1" time="18.592" errors="0" timestamp="2021-10-20T22:56:23 IDT">
  <testcase classname="Tests.ChatOnLine" name="ChatOnLineTest" time="11.151">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at Japanika.Functions.IdentifyChat(Functions.java:21)
at Tests.ChatOnLine.ChatOnLineTest(ChatOnLine.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ChatOnLineTest -->
</testsuite> <!-- Test1 -->
